test-alerts.cpp

```diff
@@ -5,4 +5,35 @@
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
```

Comment:
Good set of test values 👍 
In range value should also be tested, ex 25

Reply:
Done (yes)